lib/search/query_parameters.rb

```diff
@@ -95,11 +95,11 @@ def model_variants
   private
 
     def determine_if_quoted_phrase
-      if @query =~ QUOTED_STRING_REGEX
-        @is_quoted_phrase = true
-      else
-        @is_quoted_phrase = false
-      end
+      @is_quoted_phrase = if @query =~ QUOTED_STRING_REGEX
```

Comment:
```suggestion
      @is_quoted_phrase = @query =~ QUOTED_STRING_REGEX
```

Reply:
Oh good spot - updated.

Reply:
Turns out if was more complicated than I thought. Should _actually_ be fixed now.